app/models/concerns/turbo/broadcastable.rb

```diff
@@ -51,15 +51,15 @@ module ClassMethods
     #     belongs_to :board
     #     broadcasts_to ->(message) { [ message.board, :messages ] }, inserts_by: :prepend, target: "board_messages"
     #   end
-    def broadcasts_to(stream, inserts_by: :append, target: broadcast_target_default)
-      after_create_commit  -> { broadcast_action_later_to stream.try(:call, self) || send(stream), action: inserts_by, target: target }
+    def broadcasts_to(stream, inserts_by: :append, target: broadcast_target_default, remove_if_present: nil)
+      after_create_commit  -> { broadcast_action_later_to stream.try(:call, self) || send(stream), action: inserts_by, target: target, remove_if_present: remove_if_present.nil? ? nil : remove_if_present.try(:call, self) || send(remove_if_present) }
```

Comment:
If we want this behaviour to be there always, it would be this:
```ruby
def broadcasts_to(stream, inserts_by: :append, target: broadcast_target_default)
      after_create_commit  -> { broadcast_action_later_to stream.try(:call, self) || send(stream), action: inserts_by, target: target, remove_if_present: self }
end
```
To make it opt-out instead of opt-in
```ruby
def broadcasts_to(stream, inserts_by: :append, target: broadcast_target_default, remove_if_present: :itself)
      after_create_commit  -> { broadcast_action_later_to stream.try(:call, self) || send(stream), action: inserts_by, target: target, remove_if_present: remove_if_present.blank? ? nil : remove_if_present.try(:call, self) || send(remove_if_present) }
```

with a similar change to `#broadcast`